clients/spark/core/src/main/hadoop2/scala/io/treeverse/clients/conditional/S3ClientBuilder.scala

```diff
@@ -29,9 +30,14 @@ object S3ClientBuilder extends io.treeverse.clients.S3ClientBuilder {
         )
       } else None
 
+    val builder = AmazonS3ClientBuilder
+      .standard()
+      .withPathStyleAccessEnabled(hc.getBoolean("fs.s3a.path.style.access", true))
```

Comment:
can you make `"fs.s3a.path.style.access"` a constant too? (to keep things unified)